code/game/objects/items/kitchen.dm

```diff
@@ -89,8 +89,12 @@
 	var/bayonet = FALSE	//Can this be attached to a gun?
 	custom_price = 30
 
-/obj/item/kitchen/knife/Initialize()
+/obj/item/kitchen/knife/ComponentInitialize()
 	. = ..()
+	set_butchering()
+
+	///Adds the butchering component, used to override stats for special cases
+/obj/item/kitchen/knife/proc/set_butchering()
```

Comment:
This proc is unnecesarry.

Can just add the component in the `ComponentInitialize` instead of making an unnecesarry proc which is just one line.

Reply:
Yup, I have reverted that, should be less spaghetti now.